env/app_vars.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/sh
+>.env
+echo "# Vault Secrets" >>.env
+echo "CKEDITOR_ENVIRONMENT_ID=${CKEDITOR_ENVIRONMENT_ID}" >>.env
+echo "CKEDITOR_SECRET_KEY=${CKEDITOR_SECRET_KEY}" >>.env
+echo "CKEDITOR_UPLOAD_URL=${CKEDITOR_UPLOAD_URL}" >>.env
+echo "EDX_API_CLIENT_ID=${EDX_API_CLIENT_ID}" >>.env
+echo "EDX_API_CLIENT_SECRET=${EDX_API_CLIENT_SECRET}" >>.env
+echo "EMBEDLY_KEY=${EMBEDLY_KEY}" >>.env
+echo "MAILGUN_KEY=${MAILGUN_KEY}" >>.env
+echo "OCW_NEXT_SEARCH_WEBHOOK_KEY=${OCW_NEXT_SEARCH_WEBHOOK_KEY}" >>.env
+echo "OCW_WEBHOOK_KEY=${OCW_WEBHOOK_KEY}" >>.env
+echo "OLL_API_CLIENT_ID=${OLL_API_CLIENT_ID}" >>.env
+echo "OPENSEARCH_HTTP_AUTH=${OPENSEARCH_HTTP_AUTH}" >>.env
+echo "SECRET_KEY=${SECRET_KEY}" >>.env
+echo "SENTRY_DSN=${SENTRY_DSN}" >>.env
+echo "SOCIAL_AUTH_OL_OIDC_SECRET=${SOCIAL_AUTH_OL_OIDC_SECRET}" >>.env
+echo "STATUS_TOKEN=${STATUS_TOKEN}" >>.env
+echo "TIKA_ACCESS_TOKEN=${TIKA_ACCESS_TOKEN}" >>.env
+echo "YOUTUBE_DEVELOPER_KEY=${YOUTUBE_DEVELOPER_KEY}" >>.env
+echo "POSTHOG_PROJECT_API_KEY=${POSTHOG_PROJECT_API_KEY}" >>.env
+echo "POSTHOG_PERSONAL_API_KEY=${POSTHOG_PERSONAL_API_KEY}" >>.env
+echo >>.env
+echo "# Heroku Vault Static Vars" >>.env
+echo "ALLOWED_HOSTS=["*"]" >>.env
+echo "AWS_STORAGE_BUCKET_NAME=ol-mitlearn-app-storage-qa" >>.env
+echo "CORS_ALLOWED_ORIGIN_REGEXES=['^.+ocw-next.netlify.app$']" >>.env
+echo "CSAIL_BASE_URL=https://cap.csail.mit.edu/" >>.env
+echo "CSRF_COOKIE_DOMAIN=.rc.learn.mit.edu" >>.env
+echo "EDX_API_ACCESS_TOKEN_URL=https://api.edx.org/oauth2/v1/access_token" >>.env
+echo "EDX_API_URL=https://api.edx.org/catalog/v1/catalogs/10/courses" >>.env
+echo "EDX_PROGRAMS_API_URL=https://discovery.edx.org/api/v1/programs/" >>.env
+echo "MICROMASTERS_CATALOG_API_URL=https://micromasters.mit.edu/api/v0/catalog/" >>.env
+echo "MICROMASTERS_CMS_API_URL=https://micromasters.mit.edu/api/v0/wagtail/" >>.env
+echo "MITOL_ADMIN_EMAIL=cuddle-bunnies@mit.edu" >>.env
+echo "MITOL_AXIOS_BASE_PATH=https://rc.learn.mit.edu" >>.env
+echo "MITOL_DEFAULT_SITE_KEY=micromasters" >>.env
+echo "MITOL_EMAIL_PORT=587" >>.env
+echo "MITOL_EMAIL_TLS=True" >>.env
+echo "MITOL_ENVIRONMENT=local" >>.env
+echo "MITOL_FROM_EMAIL=MITOpen <mitopen-support@mit.edu>" >>.env
+echo "MITOL_FRONTPAGE_DIGEST_MAX_POSTS=10" >>.env
+echo "MITOL_USE_S3=True" >>.env
+echo "MITOL_NOTIFICATION_EMAIL_BACKEND=anymail.backends.mailgun.EmailBackend" >>.env
+echo "MITPE_BASE_URL=https://professional.mit.edu/" >>.env
+echo "MITX_ONLINE_BASE_URL=https://mitxonline.mit.edu/" >>.env
+echo "MITX_ONLINE_COURSES_API_URL=https://mitxonline.mit.edu/api/v2/courses/" >>.env
+echo "MITX_ONLINE_LEARNING_COURSE_BUCKET_NAME=mitx-etl-mitxonline-production" >>.env
+echo "MITX_ONLINE_PROGRAMS_API_URL=https://mitxonline.mit.edu/api/v2/programs/" >>.env
+echo "NEW_RELIC_LOG=stdout" >>.env
+echo "NODE_MODULES_CACHE=False" >>.env
```

Comment:
I'm not sure about the exact implications of this but I suspect it's not something we want locally.